src/requirements/Form/FormForm.tsx

```diff
@@ -30,22 +33,33 @@ const FormForm = ({ baseFieldPath }: RequirementFormProps) => {
         value: form.id,
       })) ?? []
 
+  const handleFormAdded = (formId: number) => {
+    mutate().then(() => {
```

Comment:
I think we don't need to call `mutate` here, we already mutate the cache directly inside `useCreateForm` ;)